### PR TITLE
#40 teacher profile chris koetzee

### PR DESCRIPTION
**Pull request by ChrisKoetzee**
## Naming Rules

#40TeacherProfileChrisKoetzee

## Description

A profile page was made for the teacher so that they could select a student and add a grade into the students table. Various components was made and imported into this profile page which could be re-used on the student info page. I also update the text on the home page with the given Vision and Mission. Once the tables and endpoints are completed we then will integrate the front end to the back end. I also made an image folder which I then included the pictures of the staff that was given to us. With regard to the Donations page as well as the Newsletter page I have linked it up already and the code just needs to be slotted in.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

#40 

## 📸 Screens
Screenshot (551)
hots

Kindly attach screenshots if available
Screenshot (552)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [x] I have updated any documentation

### CODE REVIEW

**Comment by Craques** [comment]
Looks good, may not need to upload teacher images at the moment for POPIA reasons

**Review by shafiekdavids — APPROVED** [review]

File: client/src/pages/TeacherProfile.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import Template from "../components/Template";
+import { Container, Col, Row, Table, Card } from "react-bootstrap";
+import SelectStudent from "../components/SelectStudent";
+import UpdateGrade from "../components/UpdateGrade";
+import ModalPopUp from "../components/ModalPopUp";
+
+const TeacherProfile = () => {
```
Comment (Craques) [inline client/src/pages/TeacherProfile.js:7]: We may not need this page

**Comment by Craques** [comment]
This PR may be too large, you can keep it limited to the functionality required to add grades

**Review by Craques — APPROVED** [review]

**Review by shafiekdavids — APPROVED** [review]

**Review by mfukechristian — COMMENTED** [review]
approved

**Review by mfukechristian — COMMENTED** [review]
approved

**Pull request merged**